root/io/filemerger/execTestMultiMerge.C

```diff
@@ -71,6 +71,25 @@ int testSimpleFile(const char *filename, Long64_t entries, Int_t compSetting, Lo
 int execTestMultiMerge()
 {
    Int_t result = 0;
+   int hsimpleFTolerance = 6; // 5 for non fst builds
+#ifdef R__HAS_LZ4
```

Comment:
Those seems to be the number if/when lz4 becomes the default.
We actually need to create *new* files  compressed explicitly with lz4 and add new tests calls using those.
(We also probably need to add *new* files compressed explicitly with zlib and add new test calls to distinguish them for the 'default' generated files.